mmm/describe/describe.py

```diff
@@ -467,13 +468,33 @@ def describe_mmm_training(
     )
     cost_per_target_hat = 1.0 / roi_hat
 
+    media_effect_df = get_media_effect_df(data_to_fit, media_effect_hat)
+
     _dump_posterior_metrics(
         results_dir,
-        get_media_effect_df(data_to_fit, media_effect_hat),
+        media_effect_df,
         get_roi_df(data_to_fit, media_effect_hat, roi_hat),
         get_cost_per_target_df(data_to_fit, media_effect_hat, roi_hat, cost_per_target_hat),
     )
 
+    try:
```

Comment:
If this fails only when we can't write to the file system, I'd remove the exception handler.

Unless I'm missing something, this doesn't seem like an exception we should be handling.

Reply:
I had some errors when the parsing of data to generate summary, so I'm being (maybe over) cautious to not crash the whole process if generating summary fails. Should be an easy fix when that happens, so I'll remove this try catch.